ui/app/AppLayouts/Communities/panels/OverviewSettingsFooter.qml

```diff
@@ -46,7 +45,6 @@ Control {
         id: mainGrid
         columnSpacing: 16
         rowSpacing: 16
-        visible: root.isPromoteSelfToControlNodeEnabled
```

Comment:
Removed this property; it was just hiding the contentItem! We know control the footer visibility from outside, as it should be

Reply:
This also completely broke the SB page